src/types/types.ts

```diff
@@ -163,4 +163,29 @@ export type SafeCreationResponse = {
   status: boolean;
   safeId: string | null;
   error: Error | null
+}
+
+export type SecretSafe = {
+  seedPhrase: string | null,
+  privateKey: string | null,
+  keyStore: string | null
+}
+
+export type Instructions = {
+  softwareWallet: string | null,
+  hardwareWallet: string | null,
+}
+export interface CryptoSafe {
+  walletStoreType: {
+   data: SecretSafe | Instructions
+  }
+}
+
+export interface GenericSafe {
+  data: any
+}
+
+export type SafeStorage = {
+  safeType: number
```

Comment:
I think we discussed removing the enum type as we can directly check the safe type.